atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -144,6 +168,35 @@ const CrashesByMode = () => {
     datasets: !!chartData && createTypeDatasets(),
   };
 
+  const StyledDiv = styled.div`
+    .year-total-div {
+      color: ${colors.dark};
+      background: ${colors.buttonBackground} 0% 0% no-repeat padding-box;
+      border-radius: 4px;
+      border-style: none;
+      opacity: 1;
+    }
+
+    .mode-label-div:hover {
+      cursor: pointer;
+      color: ${colors.buttonBackground};
+      background: dimgray 0% 0% no-repeat padding-box;
+      border-radius: 4px;
+      border-style: none;
+    }
+
+    .mode-label-text {
+      display: none;
+      @media (min-width: 576px) {
```

Comment:
This breakpoint should probably live in an import file so we can ensure consistent breakpoints. Can you either address that in this PR or create a new issue and label it `Tech Debt`?